scripts/webpack.config.js

```diff
@@ -145,10 +149,10 @@ function getConfig(isReactExternalized) {
         stats,
     };
 
+    config.plugins.push(Translations);
```

Comment:
apparently was only build the translations via dev and not in prod, the bundles folder has to be there when starting storybook or publishing so moved it outside of the `if`